packages/vscode-extension/package.json

```diff
@@ -46,6 +46,23 @@
         "configuration": "./cypher-language-configuration.json"
       }
     ],
+    "grammars": [
```

Comment:
are we interested in having this highlighting by default as well (instead of/in addition to the semantic highlighting)?

Reply:
This is already on by default. The semantic highlighting just overrides it